crates/ln-dlc-node/src/node/mod.rs

```diff
@@ -7,6 +7,7 @@ use crate::ln::EventHandler;
 use crate::ln::TracingLogger;
```

Comment:
🙃 the commit message actually says something different than what the change entails 😅 

"move helper function out of mod" would be correct 🙃

Reply:
in-out, same same but different